Google.Cloud.EntityFrameworkCore.Spanner.Tests/MigrationTests/GenerateCreateScriptTest.cs

```diff
@@ -42,7 +41,91 @@ public void Generate_Create_Script()
         {
             using var db = new MockMigrationSampleDbContext();
             var generatedScript = db.Database.GenerateCreateScript();
-            var script = File.ReadAllText("MigrationTests/SampleMigrationDataModel.sql");
+            var script = @"CREATE TABLE `Singers` (
```

Comment:
Is this now inlined to avoid the license check or is it for some other reason?

Reply:
Yes this is to avoid the license header check.